Update readme with how to train offline and fix BPE command (#15897)
huggingface/transformers: examples/research_projects/codeparrot/README.md

python scripts/codeparrot_training.py --help
```

> If you would like to train the model offline, you can download the dataset locally:

Reviewer: ```suggestion
Instead of streaming the dataset from the hub you can also stream it from disk. This can be helpful for long training runs where the connection can be interrupted sometimes. To stream locally you simply need to clone the datasets and replace the dataset name with their path. In this example we store the data in a folder called `data`: 
```

Author: Way clearer, thank you!!

```bash

> git lfs install
> git clone https://huggingface.co/datasets/lvwerra/codeparrot-clean-train
> git clone https://huggingface.co/datasets/lvwerra/codeparrot-clean-valid

Reviewer: ```suggestion
git lfs install
mkdir data
git -C "./data" clone https://huggingface.co/datasets/lvwerra/codeparrot-clean-train
git -C "./data" clone https://huggingface.co/datasets/lvwerra/codeparrot-clean-valid
```

```

> And then point the datasets loader to those directories:

Reviewer: ```suggestion
And then pass the paths to the datasets when we run the training script:
```

```bash
accelerate launch scripts/codeparrot_training.py \
--model_ckpt lvwerra/codeparrot-small \

> --dataset_name_train /path/to/codeparrot-clean-train \
> --dataset_name_valid /path/to/codeparrot-clean-valid \

Reviewer: ```suggestion
--dataset_name_train ./data/codeparrot-clean-train \
--dataset_name_valid ./data/codeparrot-clean-valid \
```
